applications/graphs/urls.py

```diff
@@ -6,6 +6,7 @@
 
 	url(r'^graphs/$', views.graphs_page, name='graphs'),
 	url(r'^graphs/(?P<graph_id>[^/]+)$', views.graph_page, name='graph'),
+	url(r'^graph/(?P<graph_id>[^/]+)$', views.graph_page, name='graph_by_id'),
```

Comment:
@melvin15may There is no need to add a new endpoint. You can still make a distinction from `/graphs` and `\groups` endpoint.